src/github.com/matrix-org/rageshake/submit.go

```diff
@@ -41,10 +41,12 @@ type submitServer struct {
 }
 
 type payload struct {
-	Text      string     `json:"text"`
-	Version   string     `json:"version"`
-	UserAgent string     `json:"user_agent"`
-	Logs      []logEntry `json:"logs"`
+	Text      string            `json:"text"`
+	AppName   string            `json:"app"`
+	Version   string            `json:"version"`
+	UserAgent string            `json:"user_agent"`
+	Logs      []logEntry        `json:"logs"`
+	Data      map[string]string `json:"data"`
```

Comment:
Please update the readme then: clarity trumps all, and at the moment it's a bit ambiguous:

> arbitrary name/value pairs